database/migrations/2021_05_25_135505_create_gadgets_users_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateGadgetsUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('gadgets_users', function (Blueprint $table) {
```

Comment:
Deze tabel heeft een `amount` over de hoeveelheid gadgets die een user heeft. Dus dat maakt het een één op één relatie. Dan moet de naam van de tabel ook `gadget_users` zijn ipv `gadgets_users`.

Reply:
Door de lookup table is het een veel-op-veel relatie. Als voorbeeld: een Alarm gadget kan zowel van user Karel als van user Bert zijn. Maar Karel moet bijvoorbeeld ook een Drone kunnen hebben,

Reply:
De amount komt alleen terug wanneer ik besluit om Karel bijvoorbeeld 3 Alarms te geven. Dat moet wel bijgehouden worden. Om hier aparte entries voor te maken lijkt me onverstandig. Vandaar op deze manier.